python/py-blis/Portfile

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem                                      1.0
+PortGroup           python                      1.0
+PortGroup           github                      1.0
```

Comment:
this whitespace usage (between portgroup and version) is a bit unconventional

Reply:
I like things to line up in columns, and I guess I cut and pasted this from a port where I also use a long named one....

Reply:
I've reduced some of the space.